test/os.cc

```diff
@@ -0,0 +1,44 @@
+#include "eventuals/os.h"
+
+#include <limits.h>
+
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+
+#if defined(__linux__) || defined(__MACH__)
+
+using eventuals::Thread;
```

Comment:
```suggestion
namespace eventuals::test {
namespace {

#if defined(__linux__) || defined(__MACH__)
```
See other tests so we're consistent!